src/components/dashboard/scrim-creator.tsx

```diff
@@ -182,6 +195,23 @@ export function ScrimCreationForm({
             </FormItem>
           )}
         />
+        <FormField
+          control={form.control}
+          name="replayCode"
+          render={({ field }) => (
+            <FormItem>
+              <FormLabel>Scrim Replay Code</FormLabel>
+              <FormControl>
+                <Input placeholder="Replay Code" className="w-28" {...field} />
+              </FormControl>
+              <FormDescription>
+                This is the replay code of the first map. You can edit the
+                replay codes after the scrim is created.
+              </FormDescription>
+              <FormMessage />
+            </FormItem>
+          )}
+        />
```

Comment:
I do not like how this looks in the UI. I think replay codes should not be prompted at first. We can add a hint on the scrim page to tell the user that they can add a replay code but I think the initial creation form should be as minimal as possible. I would remove this before merging.